docs/src/components/Footer/index.js

```diff
@@ -34,7 +34,7 @@ const EmailSubscription = () => {
         subscribeNewsletter && (
           <div className="margin-vert--md">
             <a
-            className="doc-button doc-button-primary doc-button-curved  doc-button-lg"
+            className="doc-button doc-button-primary doc-button-curved doc-button-lg"
```

Comment:
This change is not required I believe.

Reply:
I have removed extra space

Reply:
Though not required, it is better to keep a single space separating each class 😊